java/simulated_ble/README.md

```diff
@@ -0,0 +1,306 @@
+# Azure IoT Edge BLE Data Converter Module
+
+## Overview
+
+This tutorial showcases how one might build a module for Azure IoT Edge in Java.
+
+In this tutorial, we will walk through environment setup and how to write a [BLE](https://en.wikipedia.org/wiki/Bluetooth_Low_Energy) data converter module using the latest Azure IoT Edge packages.
```

Comment:
maybe change "latest Azure IoT Edge packages" to "latest Azure IoT Edge Maven packages"?